src/esp/assets/FRLInstanceMeshData.h

```diff
@@ -43,10 +50,19 @@ class FRLInstanceMeshData : public GenericInstanceMeshData {
 
   virtual Magnum::GL::Mesh* getMagnumGLMesh() override;
 
+  const std::vector<vec3f> get_vbo();
+  const std::vector<int> get_ibo();
+
  protected:
   std::vector<vec4f> cpu_vbo;
   std::vector<vec3uc> cpu_cbo;
 
+  std::vector<vec3f>* cpu_vbo_3   = nullptr;
+  std::vector<uint32_t>* tri_ibo  = nullptr;
+  //std::vector<int>* tri_ibo_i     = nullptr;
+  std::vector<float>* cbo_float   = nullptr;
```

Comment:
All these should be unique pointers or shared pointers (depending on if they are needed by other classes or not).
That way you don't have to delete them yourself.